theunderground/movies.py

```diff
@@ -81,6 +91,16 @@ def add_movie():
                 # Now that we've inserted the movie, we can properly move it.
                 save_movie_data(db_movie.movie_id, thumbnail_data, movie_data)
 
+                # Save the DS movie if it exists to /dsmov/{get_movie_path}/{movie_id}.enc
+                # NOT on s3
+
+                if dsmovie:
+                    dsmovie_dir = get_movie_path(db_movie.movie_id)
+                    dsmovie_path = f"{dsmovie_dir}/{db_movie.movie_id}.enc"
+
+                    with open(dsmovie_path, "wb") as f:
+                        f.write(dsmovie_data)
+
```

Comment:
I would like S3 support here. The path is the same, so reference what I did elsewhere on what to do.

Reply:
the path isnt the same. it goes to /room-assets/dsmov/??/??.enc

Reply:
My fault. You can do 
```py
dsmovie_path.split("/", 1)[1]
```
to get the path without `room-assets`.

Reply:
i meant thats where it goes on the server itself. s3 uploading currently goes to movies folder as the root, not room-assets.
if i did dsmov, it would go to (s3 equivalent of) /room-assets/movies/dsmov/??/ where its meant to go to /room-assets/dsmov

Reply:
resolved in c7fa773